apache-airflow-mixin/panels.libsonnet

```diff
@@ -0,0 +1,134 @@
+        + g.panel.timeSeries.panelOptions.withDescription('The number of errors from trying to parse DAG files in an Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('none'),
+
+      slaIssuesPanel:
+        commonlib.panels.generic.timeSeries.base.new('SLA misses', targets=[
+          signals.overview.taskSlaMisses.asTarget() { interval: '2m' },
+        ])
+        + g.panel.timeSeries.panelOptions.withDescription('The number of Service Level Agreement misses for any DAG runs in the Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('none'),
+
+      taskFailuresPanel:
+        commonlib.panels.generic.timeSeries.base.new('Task failures', targets=[
+          signals.overview.taskFailures.asTarget() { interval: '2m' },
+        ])
+        + g.panel.timeSeries.panelOptions.withDescription('The overall task instances failures for an Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('none'),
+
+      dagSuccessDurationPanel:
+        commonlib.panels.generic.timeSeries.base.new('DAG success duration', targets=[
+          signals.overview.dagAvgSuccessDuration.asTarget() { interval: '2m' },
+        ])
+        + g.panel.timeSeries.panelOptions.withDescription('The average time taken for recent successful DAG runs by DAG ID in the Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('s')
+        + g.panel.timeSeries.options.legend.withPlacement('right')
+        + g.panel.timeSeries.options.tooltip.withMode('multi'),
+
+      dagFailedDurationPanel:
+        commonlib.panels.generic.timeSeries.base.new('DAG failed duration', targets=[
+          signals.overview.dagAvgFailedDuration.asTarget() { interval: '2m' },
+        ])
+        + g.panel.timeSeries.panelOptions.withDescription('The average time taken for recent failed DAG runs by DAG ID in the Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('s')
+        + g.panel.timeSeries.options.legend.withPlacement('right')
+        + g.panel.timeSeries.options.tooltip.withMode('multi'),
+
+      taskDurationPanel:
+        g.panel.barGauge.new('Task duration')
+        + g.panel.barGauge.queryOptions.withTargets([
+          signals.overview.taskAvgDuration.asTarget() { interval: '2m' },
+        ])
+        + g.panel.barGauge.options.withOrientation('horizontal')
+        + g.panel.barGauge.panelOptions.withDescription('The average time taken for recent task runs by Task ID in the Apache Airflow system.')
+        + g.panel.barGauge.standardOptions.withUnit('s'),
+
+      taskCountSummaryPanel:
+        g.panel.pieChart.new(
+          'Task count summary'
+        )
+        + g.panel.pieChart.queryOptions.withTargets([
+          signals.overview.taskFinishTotalSum.asTarget() { interval: '2m' },
+        ])
+        + g.panel.pieChart.options.legend.withPlacement('right')
```

Comment:
Should this also use a table legend, i.e.: `+ g.panel.timeSeries.options.legend.withDisplayMode('table')`?